### ISSUE TITLE
Website first page example

### ISSUE THREAD

**Issue by omidb** [ReactiveMongo/reactivemongo-site#19]
Hi, first page example is not complete, some of 'import' are not correct because if you follow the exact think you will get error!

`Iteratee` and `find` are not there

**Comment by cchantep** [comment]
Hi,

Can you precise which import is wrong for you?

**Comment by omidb** [comment]
with the current imports it doesn't find `Iteratee` and `find`

``` scala
"org.reactivemongo" %% "reactivemongo" % "0.11.0"
```

**Comment by cchantep** [comment]
Which import? Which code example?

**Comment by omidb** [comment]
there is no import that contains `Iteratee` and `find`!

``` scala
import scala.concurrent.Future
import scala.concurrent.ExecutionContext.Implicits.global

import play.api.libs.iteratee.Iteratee

import reactivemongo.bson.BSONDocument
import reactivemongo.api.collections.bson.BSONCollection

def listDocs(collection: BSONCollection) = {
  // Select only the documents which field 'firstName' equals 'Jack'
  val query = BSONDocument("firstName" -> "Jack")
  // select only the fields 'lastName' and '_id'
  val filter = BSONDocument(
    "lastName" -> 1,
    "_id" -> 1)

  /* Let's run this query then enumerate the response and print a readable
   * representation of each document in the response */
  collection.
    find(query, filter).
    cursor[BSONDocument].
    enumerate().apply(Iteratee.foreach { doc =>
      println(s"found document: ${BSONDocument pretty doc}")
    })

  // Or, the same with getting a list
  val futureList: Future[List[BSONDocument]] =
    collection.
      find(query, filter).
      cursor[BSONDocument].
      collect[List]()

  futureList.map { list =>
    list.foreach { doc =>
      println(s"found document: ${BSONDocument pretty doc}")
    }
  }
}
```

**Comment by omidb** [comment]
it has been solved in 11d76d9881a7542d6cd44ba37446e85153531bfb
